Client.Core/CategoryTraceFilter.cs

```diff
@@ -0,0 +1,56 @@
+using System.Diagnostics;
+using System.Linq;
+
+namespace InfluxDB.Client.Core
+{
+    public class CategoryTraceFilter : TraceFilter
```

Comment:
Please rename to `InfluxDBTraceFilter`:

```suggestion
        /// <summary>
    /// The <see cref="InfluxDBTraceFilter"/> is used to filter client trace messages by category.
    /// </summary>
    public class InfluxDBTraceFilter : TraceFilter
```